exporter/collector/config.go

```diff
@@ -72,8 +72,8 @@ type MetricConfig struct {
 }
 
 // ResourceMapping defines mapping of resources from source (OpenCensus) to target (Google Cloud).
+// In order to use these resource mappings, the cloud.platform attribute must be set to gcp_custom.
```

Comment:
This now requires the cloud.platform to be gcp_custom.  An alternative would be to add a cloud_platform field in ResourceMapping that allows telling the exporter which cloud platform to apply it to.

Reply:
I prefer the second option so we don't need a magic string